src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -483,24 +530,30 @@ g.test('it_is_invalid_to_use_a_resolve_target_with_mipmap_level_count_greater_th
     };
 
     t.tryRenderPass(false, descriptor);
-  }
-);
-
-g.test('it_is_invalid_to_use_a_resolve_target_whose_usage_is_not_RENDER_ATTACHMENT').fn(async t => {
-  const multisampledColorTexture = t.createTexture({ sampleCount: 4 });
-  const resolveTargetTexture = t.createTexture({
-    usage: GPUTextureUsage.COPY_SRC | GPUTextureUsage.COPY_DST,
   });
 
-  const colorAttachment = t.getColorAttachment(multisampledColorTexture);
-  colorAttachment.resolveTarget = resolveTargetTexture.createView();
+g.test('color_attachments_RENDER_ATTACHMENT_texture_usage')
```

Comment:
perhaps `color_attachments,usage`

Reply:
How about using **color_attachments,resolveTarget_usage**?

Reply:
Oh yes, missed that this was for the resolve target. Then it can be grouped with the others, so `resolveTarget,usage`.